workspaces/redhat-resource-optimization/README.md

```diff
@@ -1,8 +1,146 @@
 # Welcome to the Resource Optimization plugin workspace
 
-Follow these links to learn more about this plugin:
+# Resource Optimization
 
-1. [Front-end](./plugins/redhat-resource-optimization/README.md)
-2. [Backend-end](./plugins/redhat-resource-optimization-backend/README.md)
-3. [Common](./plugins/redhat-resource-optimization-common/README.md)
-4. [RBAC](./docs/rbac.md)
+Welcome to the Resource Optimization plugin!
+
+Based on [Project Koku](https://github.com/project-koku/koku), the Resource Optimization plugin allows users to visualize usage trends and receive optimization recommendations for workloads running on OpenShift clusters.
```

Comment:
i do not see how this project relates to koku. we are an RHDH plugin and we pull info from redhat insights